test.js

```diff
@@ -78,6 +78,14 @@ describe('cpFile()', function () {
 			cb();
 		});
 	});
+
+	it('should not copy directories', function (cb) {
+		cpFile('node_modules', 'tmp', function (err) {
+			assert(!err, err);
+			assert(!fs.existsSync('tmp'));
```

Comment:
It's strange but `fs.existsSync` [will be deprecated](http://nodejs.org/api/fs.html#fs_fs_existssync_path).
Personally, I would use:
```
assert.throws(fs.statSync.bind(fs, 'tmp'), /ENOENT/);
```